### ISSUE TITLE
Add more info on Parity Signer with new version

### ISSUE THREAD

**Issue by laboon** [w3f/polkadot-wiki#2987]
Specifically let's include instructions for adding new parachains with latest version of Parity Signer.

From watercooler:
UPDATE: Parity Signer v5.0.0 is available in Apple Store! Remove the current version of Signer before installing the new one! Make sure you have your mnemonics!!

You can also use this extension specifically designed for Signer:
https://parity.link/signer-companion
And you can update the metadata from here when there's a runtime upgrade (v5 has the latest metadata):
https://metadata.parity.io/
For the moment it has metadata only for Polkadot, Kusama and Westend. You can add custom chains using this crate:
https://github.com/paritytech/parity-signer/tree/master/rust/generate_message
but honestly the process is beyond my understanding, so if someone could dumb it down that would be great. Or if Parity could start adding more chains in the site above.

**Comment by DrW3RK** [comment]
https://wiki.polkadot.network/docs/learn-account-generation#parity-signer
